infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/LogReplicationTransportSelectorPlugin.java

```diff
@@ -0,0 +1,12 @@
+package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+/**
+ * Log replication allows for any transport layer to be plugged in.
+ * This interface must be implemented to give the name of the property, in the plugin config file, which
```

Comment:
Is this comment completed? Would you mind to complete it if it isn't?

Reply:
done! Thanks for catching this!